src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': person.slug === slug })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
```

Comment:
Consider using destruction assignment for `person` object

Reply:
Done! Thanks for the comment)